### ISSUE TITLE
Chit Chats Webhook?

### ISSUE THREAD

**Issue by tristant2** [chitchats/chitchats-api-doc#68]

**Comment by gordon-leung** [comment]
No, the API does not currently support webhooks. I'm assuming you're referring to tracking updates?

Tracking webhooks has been added to our backlog for implementation. We'll share an update here as soon as the timeline is confirmed.
